### PR TITLE
Installation Documents Overhaul

### PR DESCRIPTION
**Pull request by KzBoy**
Cleaned up a lot of missing or confusing documentation regarding the install process.

Added instructions for Windows that was missing in quite a few locations,

Also overhauled the installation process itself.
Added a 'Installation Overview' page to help users know the steps to creating a working installation for their particular setup. This would replace the 'Getting Started' page.
Broke out the Installation methods into it's own page to make it easy to see what the options are and make adding any additional options easier in the future.

New install flow broken down into 5 steps (See the 'Overview' page)

This gives the user a 'map' for the install process and a place to refer to if they get lost/confused about what to do next or simply need help.

### CODE REVIEW

File: documentation/docs/install/controllerlogin.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+levels may not work correctly.
+
+For example,the screenshot below show the username chosen as `unifipoller`.
+This is the default, will be used throughout these docs.
+
+![img](../../static/img/UDM_user.png)
+
+### Non UnifiOS Controller
+
+If you are using another controller type (eg. Cloudkey or Virtual) then create a menual read-only user.
```
Comment (geoff2k) [inline documentation/docs/install/controllerlogin.md:26]:
```suggestion
If you are using another controller type (eg. Cloudkey or Virtual) then create a manual read-only user.
```

File: documentation/docs/install/controllerlogin.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+The `Email` field will be the 'username' you will need to create the config file.
+
+The Unifi controller currently requires the email be formated correctly. If you don't have your own domain try using @example.com so you don't inadvertantly give access to a random user.
```
Comment (geoff2k) [inline documentation/docs/install/controllerlogin.md:30]:
```suggestion
The Unifi controller currently requires the email be formatted correctly. If you don't have your own domain try using @example.com so you don't inadvertently give access to a random user.
```

File: documentation/docs/install/installmethod.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+id: installmethod
+title: Install Method
+---
+
+## Chosing an Install Method
+
+There are two main methods to install the Unpollor 'suite' (Unpoller and accociated programs)
```
Comment (geoff2k) [inline documentation/docs/install/installmethod.md:8]:
```suggestion
There are two main methods to install the Unpoller 'suite' (Unpoller and associated programs)
```

File: documentation/docs/install/overview.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+[Chosing an Install Method](../install/installmethod)
+
+### 2) Setup Unifi login for Unpoller
+No matter which method of installation you choose you will need to give Unpoller a way to access the infomation in your controller.
+You can setup Unpoller to get information from multiple controllers if you want, however we recommend getting one controller working before linking multiple.
+
+[Adding a login for Unpoller](unifilogin)
+
+### 3) Create Config
+The config tells Unpoller where to find the Unifi controller(s), database, and other infomation.
```
Comment (geoff2k) [inline documentation/docs/install/overview.md:29]:
```suggestion
The config tells Unpoller where to find the Unifi controller(s), database, and other information.
```

**Review by geoff2k — COMMENTED** [review]
Fixed a few minor typos.

**Comment by KzBoy** [comment]
Okie-dokie.... I think I'm finished cleaning up references to Unifi Poller for now. lol

I'll probably open an issue for someone to clean up the remaining pages. Would be a nice easy project for someone to tackle.

**Comment by davidnewhall** [comment]
I'm going to give this a looksie very soon! @PianSom You're welcome too if you have time/feel like it. <3

**Comment by PianSom** [comment]
Hi @davidnewhall - I'm very, very happy to help out with proof-reading/editing/extending, or anything else

But I am a bit busy right now, so may not have the chance to look at it this side of Xmas, sorry

**Comment by davidnewhall** [comment]
@KzBoy I appreciate your efforts, and we will get to this. I just have many projects that suck up my time and this one has been on the back burner for a while.

**Comment by davidnewhall** [comment]
reminder that synology installation page has `influxbd:latest` and it's wrong.

**Comment by KzBoy** [comment]
> @KzBoy I appreciate your efforts, and we will get to this. I just have many projects that suck up my time and this one has been on the back burner for a while.

Totally understand. I also know I didn't go about the changes is the best way that the readme indicates. Unfortunately I was having some trouble figuring it out and only had so much time to devote to it. So I figured maybe this would be better then nothing, and worst case it has to get thrown out, which isn't the end of the world. **Shrugs**

I still need to fine tune my setup. lol, It's basically just running the defaults still.

**Comment by platinummonkey** [comment]
👋 going to pick this up thanks for the effort here!

**Pull request merged**